src/auth-service/config/dbConnection.js

```diff
@@ -1,61 +1,51 @@
 const mongoose = require("mongoose");
-// const { Gstore, instances } = require("gstore-node");
-const config = require("./constants");
-// const { Datastore } = require("@google-cloud/datastore");
-
-// const gstore1 = new Gstore();
-// const gstore2 = new Gstore({ cache: true });
-// const gstore3 = new Gstore();
-
-// const datastore1 = new Datastore({
-//   namespace: "default",
-//   projectId: "airqo-250220",
-// });
-// const datastore2 = new Datastore({
-//   namespace: "cache-on",
-//   projectId: "airqo-250220",
-// });ç
-// const datastore3 = new Datastore({
-//   namespace: "staging",
-//   projectId: "airqo-250220",
-// });
-
-const URI = config.MONGO_URI;
-const db = mongoose.connection;
-
+const constants = require("./constants");
+const log = require("../utils/log");
+const URI = `mongodb://localhost/`;
+const { logElement, logText, logObject } = require("../utils/log");
+// const URI = constants.MONGO_URI;
+logElement("environment", process.env.NODE_ENV);
+logElement("the URI string", URI);
+
+//the DB connection options....
 const options = {
   useCreateIndex: true,
   useNewUrlParser: true,
   useFindAndModify: false,
+  useUnifiedTopology: true,
+  autoIndex: true,
+  poolSize: 10,
+  bufferMaxEntries: 0,
+  connectTimeoutMS: 10000,
+  socketTimeoutMS: 30000,
+  dbName: "airqo-auth-dev",
```

Comment:
I think this needs to be a variable -- using the constants.DB_NAME that varies basing on environment.